curriculum/challenges/english/25-front-end-development/review-relational-database/6724e3d6e1cb0c1fec3a8e4f.md

```diff
@@ -7,7 +7,384 @@ dashedName: review-relational-database
 
 # --description--
 
-Description placeholder
+## Introduction to Relational Databases
+
+- **Relational Databases**: Organize data into related tables made of rows and columns. Each row represents a record, and each column represents an attribute of the data.
+- **Advantages of Relational Databases**: Scalable, widely applicable across domains (e.g., healthcare, business, gaming), and structured to maintain reliable data.
+- **Common Use Cases**: Web development, inventory systems, e-commerce, healthcare, and enterprise applications.
+
+## Key Concepts
+
+- **Schema**: A relational database requires a schema that defines its structure—tables, columns, data types, constraints, and relationships.
+- **Primary Keys**: Unique identifiers for each row in a table. They are essential for data integrity and are used to relate records between tables via foreign keys.
+- **Foreign Keys**: References to primary keys in another table, used to link related data across tables.
+- **Relationships**: By connecting tables through primary and foreign keys, you can structure normalized data and perform meaningful queries.
+- **Entity Relationship Diagrams (ERDs)**: Visualize how entities (tables) relate to each other in a database schema.
+- **Data Integrity**: Enforced using keys and data types. Ensures consistency and accuracy of stored data.
+
+## SQL Basics
+
+- **Queries**: Requests to retrieve specific data from the database.
+
+```sql
+SELECT * FROM dogs WHERE age < 3;
+```
+
+- **WHERE clause**: Filter results based on conditions. Use comparison operators like `<`, `=`, `>`, etc.
+- **Select with ORDER BY**: Retrieve and sort results based on a column.
+
+```sql
+SELECT columns FROM table_name ORDER BY column_name;
+```
+
+## Table Operations
+
+- **Create a new table**:
```

Comment:
I feel like for all of these bullet points it should be the following so it matches the pattern already setup earlier.

````md
**`CREATE TABLE` Statement**: This statement is used to create a new table in a database.

```sql
example goes here
```
````

Doing a quick scan, it looks like this needs to be applied to all bullet points up till the Relational vs Non-Relational section